tests/compiler_settings/BUILD

```diff
@@ -0,0 +1,32 @@
+# Copyright 2023 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("//cc:defs.bzl", "cc_binary")
+
+licenses(["notice"])
+
+cc_binary(
+    name = "main",
+    srcs = ["main.cc"],
+    local_defines = select(
```

Comment:
Hey @fmeum 
As it appears I have not imported this PR internally - so I was trying to do so today.
Doesn't this select need a default condition? Or am I missing something. No action is required from your side

Reply:
I intentionally omitted the default condition here to verify that all CI runs match at least one of the compilers. A failure would indicate that we are missing one. But I could see this becoming a problem internally, so feel free to add it.